### DISCUSSION TITLE
with_parent_window and rwh_05

### DISCUSSION THREAD

**Discussion by ennis** [rust-windowing/winit#3228] [Q&A] [answered]
Is there a way to use `with_parent_window` with a rwh_05 handle? I'm using winit 0.29 with the `rwh_05` feature because wgpu on crates.io (0.18) hasn't yet updated to raw_window_handle 0.6, and after winit 0.29 `with_parent_window` is only available for `rwh_06`, unless I overlooked something.

**Comment by kchibisov** [comment, marked as answer]
No, but you can crate rwh_06 from 05
